tools/serverpod_cli/bin/create/copier.dart

```diff
@@ -54,11 +54,22 @@ class Copier {
     print('  ${p.join(dstDir.path, relativePath, fileName)}');
 
     var dstFile = File(p.join(dstDir.path, dstFileName));
-    var contents = srcFile.readAsStringSync();
-    contents = _replace(contents, replacements);
-    contents = _filterLines(contents, removePrefixes);
-    dstFile.createSync(recursive: true);
-    dstFile.writeAsStringSync(contents);
+    try {
```

Comment:
Why is this code needed? I it gets triggered it means that the icons isn't copied and then the fix doesn't work?

Reply:
So basically, `ico` files cannot be read in utf8 format. The previous action failed due to it.

Sample code
```dart
import 'dart:io';

Future<void> main() async {
  try {
    var a = await File('/Users/mr.minnu/favicon.ico').readAsString();
    print(a);
  } on Exception catch (e) {
    print(e.toString());
  }
}
```

Output

```console
FileSystemException: Failed to decode data using encoding 'utf-8', path = '/Users/mr.minnu/favicon.ico'
```

Reply:
Then, in it's current state this PR does not work. So I cannot approve it.

The templates are used for creating new projects with `serverpod create`. If I approved this PR, every new project would have a broken link and `serverpod create` would always output an error message. The code change you made just makes the tests pass, but doesn't fix the issue.